torch/_inductor/scheduler.py

```diff
@@ -224,6 +224,65 @@ def debug_str_extra(self):
     def is_extern(self):
         return True
 
+    def allocate(self):
```

Comment:
Deduplicate these, should be possible to share code

Reply:
yep. First, I needed to resolve the difference- I removed some lines starting here from my copy
https://github.com/pytorch/pytorch/blob/master/torch/_inductor/scheduler.py#L331

namely, V.kernel in my case was a NullKernel or something, and had no attributes or functions such as 'make_inplace'. I don't know what kernel means in this case, especially if accessed through the V object. (Unclear how it relates to my current node).  I'll see if I can figure this out, and then if using the 'make_inplace' logic would actually help make my changes more robust.  Not sure if this is relevant at all.